x-pack/plugins/apm/server/lib/transactions/spans/get_spans.ts

```diff
@@ -42,15 +41,7 @@ export async function getSpans(
           ]
         }
       },
-      sort: [{ [SPAN_START]: { order: 'asc' } }],
-      aggs: {
-        types: {
-          terms: {
-            field: SPAN_TYPE,
-            size: 100
-          }
-        }
-      }
+      sort: [{ [SPAN_START]: { order: 'asc' } }]
```

Comment:
Unrelated change: I stumbled upon this aggregation which is never used. I could make a new PR but it felt small enough to go into this one.